tests/php/bootstrap.php

```diff
@@ -16,6 +16,10 @@
  */
 define( 'TESTING_IN_JETPACK', true );
 
+// ensure we load plugins from the correct place
+define( 'WP_PLUGIN_DIR', dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) );
```

Comment:
This is not needed from my tests with latest VVV. Throws a `PHP Notice:  Constant ABSPATH already defined in /srv/www/wordpress-trunk/public_html/wp-tests-config.php on line 7` error.